spec/presenters/tree_builder_alert_profile_obj_spec.rb

```diff
@@ -0,0 +1,126 @@
+describe TreeBuilderAlertProfileObj do
+  before do
+    role = MiqUserRole.find_by(:name => "EvmRole-operator")
+    group = FactoryGirl.create(:miq_group, :miq_user_role => role, :description => "Tags Group")
+    login_as FactoryGirl.create(:user, :userid => 'tags_wilma', :miq_groups => [group])
+  end
+
```

Comment:
I deleted the ! in half of the lets. But now code is little confusing, some of them have lets, some of them does not. This does not seem to me as good approach, did I miss something? For example I left ! only in situation where I did not use the let methods before tree construction. This way we will be facing calback hell.